packages/cli/src/environments/versionControl/versionControl.controller.ee.ts

```diff
@@ -234,105 +232,4 @@ export class VersionControlController {
 			throw new BadRequestError((error as { message: string }).message);
 		}
 	}
-
```

Comment:
Shouldn't these changes be in the version control PR?

Reply:
technically this _is_ the version control PR....